lib/CFRPageActions.jsm

```diff
@@ -92,14 +95,16 @@ class PageAction {
     this.container.hidden = true;
     this._clearScheduledStateChanges();
     this.urlbar.removeAttribute("cfr-recommendation-state");
+    this.container.removeEventListener("click", this._handleClick);
+    this.urlbar.removeEventListener("focus", this._collapse);
     if (this.currentNotification) {
       this.window.PopupNotifications.remove(this.currentNotification);
       this.currentNotification = null;
     }
   }
 
   _expand(delay = 0) {
-    if (!delay) {
+    if (delay === 0 || !Number.isInteger(delay)) {
```

Comment:
If you want to keep the same code blocks unmoved, you could `if (!(delay > 0))` 😉 Probably wants a comment for that.

Alternatively just `typeof delay !== "number"` with no `= 0` default could work. Does a 0 delay actually want to clear instead of until the next tick?

Reply:
I think running immediately is fine, I don't see any point in creating any (admittedly small) overhead and we don't have any use-cases in the code that call for waiting a tick.